docs/reference/ml/ml-shared.asciidoc

```diff
@@ -537,17 +537,33 @@ the detectors in the `analysis_config`, starting at zero.
 end::detector-index[]
 
 tag::dfas-alpha[]
-Regularization factor to penalize deeper trees when training decision trees.
+Advanced configuration option. {ml} uses loss guided tree growing.
+This means that trees will grow where the regularized loss reduces
+the most. This parameter multiplies a term based on tree depth in
+the regularized loss. Higher values result in shallower trees
+and faster training times. Values should be greater than or equal
+to zero. By default, this value is calculated during hyperparameter optimization.
 end::dfas-alpha[]
 
 tag::dfas-downsample-factor[]
-The value of the downsample factor.
+Advanced configuration option. This controls the fraction of data
+that is used to compute the derivatives of the loss function for tree training.
+The lower the value the smaller the fraction of data that is used.
+Typically accuracy improves if this is set to be less than 1. However, too small
+a value may result in poor convergence for the ensemble and so require more trees.
+For more information about shrinkage, see
```

Comment:
Nit. We try to avoid expressions like "see" in the docs for being more inclusive.

```suggestion
For more information about shrinkage, refer to
```